go.mod

```diff
@@ -9,3 +9,5 @@ require (
 	google.golang.org/api v0.5.0
 	google.golang.org/grpc v1.21.0
 )
+
+replace github.com/census-instrumentation/opencensus-proto => github.com/owais/opencensus-proto v0.3.0-pr-202
```

Comment:
Will be removed once https://github.com/census-instrumentation/opencensus-proto/pull/202 is merged and 0.3.0 is released